src/database/video_music_emotion_database.py

```diff
@@ -27,11 +27,11 @@ def __init__(self):
             username=cfg['mongo_user'], password=cfg['mongo_pass'])
         self._db = self._client[cfg['mongo_db']]
 
-    def insert(self, col, song_id, video_id,
-               bpm, timbre, party, relaxed, emotions):
-        collection = self._db[col]
+    def insert(self, name, song_id, video_id,
```

Comment:
I'd change this back to col as it is the name of the collection, not the name of the database.